src/Microsoft.AspNetCore.Blazor/Components/BlazorComponent.cs

```diff
@@ -175,14 +175,21 @@ private void HandleException(Exception ex)
             Console.Error.WriteLine($"[{ex.GetType().FullName}] {ex.Message}\n{ex.StackTrace}");
         }
 
-        void IHandleEvent.HandleEvent(UIEventHandler handler, UIEventArgs args)
+        void IHandleEvent.HandleEvent(EventBinding binding, UIEventArgs args)
```

Comment:
Not keen on the name `binding` or `EventBinding`, because the word "binding" is so overloaded and it's hard to tell whose notion of "binding" this refers to.

Based on what the implementation of `EventBinding` does, might `EventHandlerInvoker` or `EventDelegateInvoker` be a clearer name?

Reply:
Good feedback. I'll go with `EventHandlerInvoker`.